### PR TITLE
Creating bar graph from covid data

### PR DESCRIPTION
**Pull request by lqueipom**

### CODE REVIEW

**Comment by Tetheta** [comment]
Starting review

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import com.google.sps.data.Cases;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@WebServlet("/chart")
+public class ChartServlet extends HttpServlet {
+
+  private Collection<Cases> covidCases;
+
```
Comment (Tetheta) [inline portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:20]: nit: extra empty line

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+@WebServlet("/chart")
+public class ChartServlet extends HttpServlet {
+
+  private Collection<Cases> covidCases;
+  
+  
+  @Override
+  public void init() {
+
```
Comment (Tetheta) [inline portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:24]: nit: extra empty line

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -204,23 +204,22 @@ function loadFavoritePlaces() {
...
+    listData.push(['State', 'Number of Cases', 'Number of Deaths']);
+    covidCases.forEach((entry) => {
+      listData.push([entry.state, entry.cases, entry.deaths]);
+    });
+    const dataset = new google.visualization.arrayToDataTable(listData);
+    
+    const parameters = {
+      chart: {
+      'width':900,
+      'height':500},
```
Comment (Tetheta) [inline portfolio/src/main/webapp/script.js:218]: nit: space after :
nit: put the } on its own line if the first { is followed by a hard return

Comment (lqueipom) [reply portfolio/src/main/webapp/script.js]: Fixed the nits, thank you for the comments!

**Pull request merged**